engine/src/main/java/org/camunda/bpm/engine/impl/util/CallableElementUtil.java

```diff
@@ -40,9 +44,38 @@ public static ProcessDefinitionImpl getProcessDefinitionToCall(VariableScope exe
     String processDefinitionKey = callableElement.getDefinitionKey(execution);
     String tenantId = callableElement.getDefinitionTenantId(execution);
 
+    return getCalledProcessDefinition(execution, callableElement, processDefinitionKey, tenantId);
+  }
+
+  public static ProcessDefinitionEntity getStaticallyBoundProcessDefinition(BaseCallableElement callableElement, String tenantId) {
+    if(callableElement.hasDynamicReferences()){
+      return null;
+    }
+
+    VariableScope emptyVariableScope = new StartProcessVariableScope();
+
+    if (callableElement.getTenantIdProvider() instanceof ElValueProvider) {
+      tenantId = callableElement.getDefinitionTenantId(emptyVariableScope);
+    }
```

Comment:
Could you detail why we don't need to call `getDefinitionTenantId` for the other providers for the tenant id?

Reply:
This was the only part that could not be easily reused from `BaseCallableElement`. Hence why I had to add a second util function in the first place. When resolving the `TenantId` for callableElements that do not have one set, the method will throw an exception. However, I agree that its not a straight forward solution here. An alternative that just came to mind was to add a method similar to `isVersionBinding` to `BaseCallableElement`.

Reply:
I'll update `getStaticallyBoundProcessDefinition` to return `ProcessDefinition`

Reply:
OK, got it. A couple more questions:

* Does it make sense to still write something more generic here to cover all static providers (since we already checked that there are no dynamic ones with `callableElement.hasDynamicReferences()`)? 
* We are reassigning an incoming method parameter `tenantId` here which is bad practice, right?
* Should we really reuse the calling process' tenant id in case the called process does not specify one? Will this resolve to the same target that the runtime will find? It sounds like the tenant id on the callable element is mandatory, is that really true? It might very well be, I am just wondering.

```suggestion
    String targetTenantId = tenantId;
    try {
      targetTenantId = callableElement.getDefinitionTenantId(emptyVariableScope);
    } catch (Exception ex) {
      // no tenant id found, let's keep using the calling process' tenant id
    }
```